docs/reference/cli-commands.md

```diff
@@ -868,6 +868,7 @@ pebble run --args myservice --port 8080 \; --hold
           --hold         Do not start default services automatically
           --http=        Start HTTP API listening on this address (e.g.,
                          ":4000") and expose open-access endpoints
+          --https=       Start HTTPS API listening on this address (e.g. ":8443")
```

Comment:
See also changes in https://github.com/canonical/pebble/pull/606/files#diff-26d4ccdee2a828c580153cbc020472b62b589e9bec6c89179c6e9f48c4a4e783R62 to give IPv4 and IPv6 examples here -- probably want to follow that.

Reply:
Fixed, thank you. Resolving.